src/com/spikes2212/Cameras/CameraController.java

```diff
@@ -7,7 +7,7 @@
 import edu.wpi.first.wpilibj.vision.USBCamera;
 
 public class CameraController {
-	public static int resolutionWidth = 100, resolutionHeight = 100;
+	public final static int resolutionWidth = 100, resolutionHeight = 100;

```

Comment:
Please make these values reasonable, check Stronghold project for the defaults we used, those should be fine